configs/310014.yaml.in

```diff
@@ -109,8 +109,9 @@ smf:
     server:
       - address: 127.0.0.4
   session:
-    - subnet: 10.45.0.1/16
-    - subnet: 2001:db8:cafe::1/48
+    - subnet: 10.45.0.0/16
+      gateway: 10.45.0.1
+    - subnet: 2001:db8:cafe::/48
```

Comment:
Seems you are missing here a "gateway: 2001:db8:cafe::1/48" in this and all other config files?

Reply:
Oops! I've fixed it in the following commit.
https://github.com/open5gs/open5gs/commit/f960047ccb1007f233775a6a5f2cdbe6f08a607f

Thank you for commenting.
Sukchan